src/data_gradients/dataset_adapters/config/data_config.py

```diff
@@ -152,7 +152,10 @@ def _fill_missing_params(self, json_dict: JSONDict):
         if self.n_classes is None:
             self.n_classes = json_dict.get("n_classes")
         if self.class_names is None:
-            self.class_names = json_dict.get("class_names")
+            class_names = json_dict.get("class_names")
+            if isinstance(class_names, dict):
```

Comment:
dict -> Mapping? (Thinking of cases when class_names can be DictConfig. Can it?)

Reply:
I don't think this will happen but why not [f4b905e](https://github.com/Deci-AI/data-gradients/pull/210/commits/f4b905ead8f00274c67efcb141aec89cd5396afe)